code/modules/mob/living/simple_animal/hostile/netherworld.dm

```diff
@@ -80,9 +80,20 @@
 	.=..()
 	START_PROCESSING(SSprocessing, src)
 
+/obj/structure/spawner/nether/examine(mob/user)
+	..()
+	if(isskeleton(user) || iszombie(user))
```

Comment:
```suggestion
	if(isskeleton(user) || iszombie(user) || iscatperson(A))
```

Reply:
lol

Reply:
This doesn't compile so I had to revert it (rip)

Reply:
```code\modules\mob\living\simple_animal\hostile\netherworld.dm:85:error: user.dna.species: undefined var```

Looks like the iscatperson() check needs to be fixed or something

Reply:
not all mobs have dna, it's not the check that is getting things wrong